tests/wgsl-errors.rs

```diff
@@ -64,12 +64,12 @@ fn invalid_integer() {
 #[test]
 fn invalid_float() {
     check(
-        "let scale: f32 = 1.1.;",
-        r###"error: expected ';', found '.'
-  ┌─ wgsl:1:21
+        "const scale: f32 = 1.1.;",
+        r###"error: expected identifier, found ';'
+  ┌─ wgsl:1:24
   │
-1 │ let scale: f32 = 1.1.;
-  │                     ^ expected ';'
+1 │ const scale: f32 = 1.1.;
+  │                        ^ expected identifier
```

Comment:
Not sure anything can be done about this.

The previous parser parsed only a single literal or construction, but we now parse a full expression because it is valid in the spec (we don't actually support most expressions, but we should) - so this becomes a field access on `1.1`.

Reply:
ah, I see - it's fine then